packages/tokens/README.md

```diff
@@ -0,0 +1,38 @@
+# @vapor/tokens
+
+Design tokens of the Vapor Design System extracted from Figma.
+
+#### What is a design token?
+
+Simply put, a design token is a stored design decision. It can be understood as a variable that holds the result of a choice. Decisions can be made about multiple different subjects, for example colors, typography, spacing, etc. Therefore, it is important to consider that design tokens come in as many types as there are design subjects.
+
+#### Objective of this package
+
+Distribute design tokens to all Coveo products that rely on the Vapor Design System through code.
+
+## Installation
+
+```bash
+npm install @vapor/tokens
```

Comment:
```suggestion
# not
pnpm install @vapor/tokens
#?
```

Reply:
not necessarily because the user of the package might not use pnpm